### ISSUE TITLE
DLL load failed with 2.0

### ISSUE THREAD

**Issue by rwhitt2049** [conda-forge/python-xxhash-feedstock#15]
Issue:

tl;dr - can't import version 2.0 from conda-forge if you only have the c++ [redistributeables](https://support.microsoft.com/en-us/help/2977003/the-latest-supported-visual-c-downloads) and not the whole of Visual Studio. Fixed by installing version 1.4.* or 2.0 from pypi

Importing xxhash 2.0.0 from conda-forge produces this error:

```python
>>> import xxhash
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Users\user\AppData\Local\Continuum\miniconda3\envs\xxhash-3.7\lib\site-packages\xxhash\__init__.py", line 1, in <module>
    from ._xxhash import (
ImportError: DLL load failed: The specified module could not be found.
```

This seems to only be an issue with python-xxhash from conda-forge explicitly as I can pip install the same version from pypi with no issue. I'm not very knowledgeable about DLL related trouble shooting, but what I was able to narrow down the following with the help from a colleague. Hopefully this is enough information to reproduce or figure out a fix as I this one is above me.

- This is an issue if you only install the redistributables instead the whole of visual studio
- It appears the the version on conda-forge is compiled slightly differently than version 1.4 and 2.0 from pypi
  - version 2.0 has the xxhash.dll specified separately from the package on conda-forge
  - version 2.0 from conda-forge is compiled against the debug version of the required dll's

Output from Dependency Walker

Python-xxhash 2.0 (getting an `ImportError: DLL load failed` from conda-forge):

It appears that python-xxhash 2.0 from conda-forge was complied against the debug dlls? Note the "D" at the end of the dll name.
 
![image](https://user-images.githubusercontent.com/4194594/90413142-b3c34080-e073-11ea-8227-2673053a88d3.png)

xxhash 1.4 from conda-forge or pypi, or version 2.0 from pypi

![image](https://user-images.githubusercontent.com/4194594/90414131-ffc2b500-e074-11ea-97b1-cc1d4dc2ec97.png)

<br/>
Environment (<code>conda list</code>):
<details>

```
$ conda list
# packages in environment at C:\Users\user\AppData\Local\Continuum\miniconda3\envs\xxhash-3.7:
#
# Name                    Version                   Build  Channel
ca-certificates           2020.6.20            hecda079_0    conda-forge
certifi                   2020.6.20        py37hc8dfbb8_0    conda-forge
openssl                   1.1.1g               he774522_1    conda-forge
pip                       20.2.2                     py_0    conda-forge
python                    3.7.8           h60c2a47_1_cpython    conda-forge
python-xxhash             2.0.0            py37h4ab8f01_0    conda-forge
python_abi                3.7                     1_cp37m    conda-forge
setuptools                49.6.0           py37hc8dfbb8_0    conda-forge
sqlite                    3.32.3               he774522_1    conda-forge
vc                        14.1                 h869be7e_1    conda-forge
vs2015_runtime            14.16.27012          h30e32a0_2    conda-forge
wheel                     0.34.2                     py_1    conda-forge
wincertstore              0.2                   py37_1003    conda-forge
xxhash                    0.8.0                he774522_1    conda-forge

```
</details>

<br/>
Details about  <code>conda</code> and system ( <code>conda info</code> ):
<details>

```
$ conda info

     active environment : xxhash-3.7
    active env location : C:\Users\user\AppData\Local\Continuum\miniconda3\envs\xxhash-3.7
            shell level : 1
       user config file : C:\Users\user\.condarc
 populated config files : C:\Users\user\.condarc
          conda version : 4.8.3
    conda-build version : 3.19.2
         python version : 3.7.1.final.0
       virtual packages : __cuda=8.0
       base environment : C:\Users\user\AppData\Local\Continuum\miniconda3  (writable)
           channel URLs : http://internal-secret-channel/channels/public/win-64
                          http:/internal-secret-channel/channels/public/noarch
                          https://conda.anaconda.org/conda-forge/win-64
                          https://conda.anaconda.org/conda-forge/noarch
                          https://repo.anaconda.com/pkgs/main/win-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/win-64
                          https://repo.anaconda.com/pkgs/r/noarch
                          https://repo.anaconda.com/pkgs/msys2/win-64
                          https://repo.anaconda.com/pkgs/msys2/noarch
          package cache : C:\Users\user\AppData\Local\Continuum\miniconda3\pkgs
                          C:\Users\user\.conda\pkgs
                          C:\Users\user\AppData\Local\conda\conda\pkgs
       envs directories : C:\Users\user\AppData\Local\Continuum\miniconda3\envs
                          C:\Users\user\.conda\envs
                          C:\Users\user\AppData\Local\conda\conda\envs
               platform : win-64
             user-agent : conda/4.8.3 requests/2.24.0 CPython/3.7.1 Windows/10 Windows/10.0.18362
          administrator : False
             netrc file : None
           offline mode : False
```
</details>

**Comment by adament** [comment]
This should be fixed by the recently released xxhash package for windows. At least I can no longer reproduce the behaviour.

**Comment by rwhitt2049** [comment]
Pruning some of my open issues and noticed this has been fixed.
